modules/drivers/usb_cam_new/AUTHORS.md

```diff
@@ -0,0 +1,9 @@
+Original Authors
```

Comment:
'usb_cam_new' is not a good folder name.

Reply:
If this is a package written by others, import to Apollo as a dependency.

Reply:
Try to use another name, such as camera_driver ?

Reply:
It's a temporary folder name to co-exist with the current "usb_cam" folder, and will rename back to "usb_cam" once the current "usb_cam" is deprecated.